scripts/cherry-pick-commits.js

```diff
@@ -52,10 +55,13 @@ const logFormat = {
 const logSplitter = ';;;';
 
 // Resolves to the most recent non-pre-release git tag.
-async function getMostRecentTag() {
+async function getMostRecentTag(mode) {
   const tags = await simpleGit.tags();
-  // Filter old independent-versioned tags and pre-releases
-  const filteredTags = tags.all.filter((tag) => /^v/.test(tag) && !/-/.test(tag));
+  // Filter old independent-versioned tags and pre-releases.
+  // If cherry-picking for a minor release, ignore patch releases to work from the latest major or minor release.
+  const filteredTags = tags.all.filter((tag) => /^v/.test(tag) && !/-/.test(tag) &&
+      (mode !== 'minor' || /\.0$/.test(tag)));
```

Comment:
can `mode !== minor` be changed to the affirmative? `mode === 'patch'`

Reply:
Do you think that's easier to read than this? I wanted to express "restrict to .0 versions for minor releases" which led me to include minor in the condition.

Reply:
could you pull this into a variable name. I think that would be clearer.

Reply:
We decided naming a variable for this would be even harder, and the other comments I added improved this.